code/modules/projectiles/medbeam.dm

```diff
@@ -0,0 +1,126 @@
+/obj/item/weapon/gun/medbeam
+	name = "Medical Beamgun"
+	desc = "Delivers medical nanites in a focused beam."
+	icon = 'icons/obj/chronos.dmi'
+	icon_state = "chronogun"
+	item_state = "chronogun"
+	w_class = 3.0
+
+	var/mob/living/current_target
+	var/last_check = 0
+	var/check_delay = 10 //Check los as often as possible, max resolution is SSobj tick though
+	var/max_range = 8
+	var/active = 0
+	var/datum/beam/current_beam = null
+
+	heavy_weapon = 1
```

Comment:
you technically can, but it has consequences--such as the gun flying out of your hands.